### PR TITLE
Add themed logo to README and update link to docs

### PR DESCRIPTION
**Pull request by AMoo-Miki**
Signed-off-by: Miki <mehranb@amazon.com>

### Description
Add themed logo to README
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by saratvemulapalli** [comment]
Thanks @AMoo-Miki for the change. Can you fix the broken link for docs?
https://github.com/opensearch-project/opensearch-plugins/blob/main/STANDARDS.md#public-documentation
It moved to: https://opensearch.org/docs/

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
```
Comment (saratvemulapalli) [inline README.md:1]: OpenSearch uses a different asset. 
Do you think having a consistent asset makes more sense?
https://opensearch.org/assets/brand/SVG/Logo/opensearch_logo_default.svg
Ref: https://raw.githubusercontent.com/opensearch-project/OpenSearch/main/README.md

Comment (AMoo-Miki) [reply README.md:1]: This themed SVG uses the "default" and "dark" modes from [OpenSearch Branding page](https://opensearch.org/brand.html) in one file to show the appropriate one to the users based on their browser's theme.

**Comment by AMoo-Miki** [comment]
> Thanks @AMoo-Miki for the change. Can you fix the broken link for docs?
> https://github.com/opensearch-project/opensearch-plugins/blob/main/STANDARDS.md#public-documentation
> It moved to: https://opensearch.org/docs/



> Thanks @AMoo-Miki for the change. Can you fix the broken link for docs?
> https://github.com/opensearch-project/opensearch-plugins/blob/main/STANDARDS.md#public-documentation
> It moved to: https://opensearch.org/docs/

Sure. I will see if there are any more changed URLs to docs.

File: README.md (hunk 1 above)
Comment (saratvemulapalli) [reply README.md:1]: Got it. Thanks for that info. It means we have to change OpenSearch repo with this new asset.

**Review by saratvemulapalli — APPROVED** [review]

**Pull request merged**